src/zino/statemodels.py

```diff
@@ -350,6 +350,7 @@ class BFDEvent(Event):
     bfdix: Optional[int] = None
     bfddiscr: Optional[int] = None
     bfdaddr: Optional[IPAddress] = None
+    neigh_rdns: Optional[str] = None
```

Comment:
According to @runborg  the name needs to be `Neigh-rDNS` ref: https://github.com/Uninett/zino/issues/199#issuecomment-2186359376
```suggestion
    Neigh-rDNS: Optional[str] = None
```

Reply:
Indeed, except this is an invalid name in Python, so this doesn't work.  We may have to look into Pydantic field aliases.  One thing is supporting them on deserialization, but I can't recall what we do for serializing to the API, I'll have to check

Reply:
This is the case for several values (`alarm_count` and `alarm_type` in AlarmEvent for instance). A simple solution would be to "zinofy" the attribute name in `model_dump_simple_attrs` by replacing `_` with `-`

Reply:
True, `model_dump_simple_attrs` exists specifically for the legacy protocol - so it should indeed replace `_` with `-` in the output (are there any attrs in the protocol that use an underscore?)

However, that still isn't enough if the *casing* is also important for the protocol clients, which makes me think a Field alias might be the way to go.  `model_dump_simple_attrs` could look up field aliases and use them if they exist, in addition to replacing underscores.

Reply:
On the other hand, question for @runborg : I cannot see that `Neigh-rDNS` is referred to anywhere else in the Zino Tcl code, not even in the Ritz tree (even when searching case insensitively).  Which again leads me to wonder: Does the casing really matter to anybody?

Reply:
Casing is a good point, if that turns out to matter then something per-field is prob required.

I'd say this outside the scope of this PR anyways, would be weird to introduce special handling of event attributes for neigh_rdns only